### PR TITLE
Add feature for the Battery Status API

### PR DESCRIPTION
**Pull request by queengooborg**
This PR adds a feature for the Battery Status API.

### CODE REVIEW

File: features/battery.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Battery Status
```
Comment (captainbrosset) [inline features/battery.yml:1]:
```suggestion
name: Battery status
```

File: features/battery.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+description: The Battery Status API allows web developers to obtain information about the device's battery, including charge level and status.
```
Comment (captainbrosset) [inline features/battery.yml:2]:
```suggestion
description: The `navigator.getBattery()` API obtains information about the device's battery, including charge level and status. Also known as the Battery API.
```

**Review by captainbrosset — DISMISSED** [review]

File: features/battery.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+name: Battery status
+description: The `navigator.getBattery()` API obtains information about the device's battery, including charge level and status. Also known as the Battery API.
+spec: https://w3c.github.io/battery/
```
Comment (ddbeck) [inline features/battery.yml:3]:
```suggestion
spec: https://w3c.github.io/battery/
caniuse: battery-status
```

File: features/battery.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+status:
+  compute_from: api.BatteryManager
```
Comment (ddbeck) [inline features/battery.yml:5]: This is not wrong exactly but, in general, prefer `compute_from` values that are the most-essential entry point(s) to the feature, what a developer would actually have to type to use it. Here that's the `navigator` API, not the `BatteryManager` interface.

(The exception is when you need to align with caniuse—where you'd use the minimal set of keys to achieve that, assuming the entry point is not sufficient—but that's not needed here.)

```suggestion
  compute_from: api.Navigator.getBattery
```

**Review by ddbeck — CHANGES_REQUESTED** [review]

**Review by ddbeck — APPROVED** [review]

**Pull request merged**